src/resources/LimitedRestorationExtension/ULC.ts

```diff
@@ -8,19 +8,11 @@ export const UlcRestorationResource: ResourceIF = {
   displayName: GetCorpFullDescription(CorpTypeCd.BC_ULC_COMPANY),
   addressLabel: 'Registered Office',
   showEmailUnderName: false,
-  filingData: [
-    // order matters - see restorationResource()
-    {
-      entityType: CorpTypeCd.BC_ULC_COMPANY,
-      filingTypeCode: FilingCodes.RESTORATION_LTD_EXTEND,
-      priority: false
-    },
-    {
-      entityType: CorpTypeCd.BC_ULC_COMPANY,
-      filingTypeCode: FilingCodes.RESTORATION_LTD_TO_FULL,
-      priority: false
-    }
-  ],
```

Comment:
Rather than overloading the restoration resource files with filing fees for multiple filing types, we're creating more resource files.  Yes, it adds duplication, but it keeps the pattern clean.